src/components/timeline/LayerActivity.svelte

```diff
@@ -194,6 +201,26 @@
     }
   }
 
+  function removeKeyDownEvent() {
+    if (browser) {
+      document.removeEventListener('keydown', onKeyDown);
+    }
+  }
+
+  function onBlur(e: FocusEvent | undefined) {
+    if (e) {
+      removeKeyDownEvent();
+    }
+  }
+
+  function onKeyDown(event: KeyboardEvent): void {
+    const { key } = event;
+
+    if (key === 'Delete' && !!selectedActivityId) {
+      dispatch('delete', selectedActivityId);
```

Comment:
This only works for me on mac if i use an external keyboard and press the delete key near fn – the other main delete key (backspace really but called delete) doesn't work. And if i'm just on my mac keyboard i can't use the normal delete key.